src/common/Microsoft.DotNet.UpgradeAssistant.Abstractions/Dependencies/IDependencyCollection.cs

```diff
@@ -10,5 +10,9 @@ public interface IDependencyCollection<T> : IEnumerable<T>
         bool Add(T item, BuildBreakRisk risk = BuildBreakRisk.None);
 
         bool Remove(T item, BuildBreakRisk risk = BuildBreakRisk.None);
+
+        HashSet<T> Additions { get; }
```

Comment:
Why is this being added? Those were explicitly hidden and I'd rather not expose them

Reply:
Coz the change to use the interface in PackageUpdaterStep could not find the Additions and Deletions. Is there a better way to expose this?

Reply:
Changed to ReadOnlyCollection